src/test/java/com/amadeus/jenkins/plugins/workflow/libs/ConfigurationRoundtripTest.java

```diff
@@ -88,14 +88,13 @@ public void testFormatDidNotChange() throws Exception {
     }
 
     private String getConfig() throws IOException {
-        String previousConfig = null;
         assertThat(globalLibraries.getLibraries()).isNotEmpty();
         File configFile = new File(j.jenkins.getRootDir(), globalLibraries.getId() + ".xml");
         assertThat(configFile).exists();
-        try (FileInputStream input = new FileInputStream(configFile)) {
-            previousConfig = IOUtils.toString(input);
-        }
-        return previousConfig;
+        String config = Files.readString(configFile.toPath(), StandardCharsets.UTF_8);
+        return config.replaceAll(
+                "<org.jenkinsci.plugins.workflow.libs.GlobalLibraries plugin=\"pipeline-groovy-lib@.*\">",
+                "<org.jenkinsci.plugins.workflow.libs.GlobalLibraries>");
```

Comment:
Recent versions of core embed the plugin version in the XML. Since this would be changing frequently as Dependabot updates the plugin BOM, strip the version to ensure test stability.